config/c-and-cpp.org

```diff
@@ -0,0 +1,11 @@
+* Additional C / C++ components
+
+If you want to use [[https://github.com/MaskRay/ccls][=ccls=]], which supports more language server features than =clangd= like overlays and semantic highlighting,
+
```

Comment:
Might be good to add that it's better to build it manually instead of using a package manager and specify explicit path.

Reply:
I didn't have a problem installing `ccls` from the AUR. I can work with you to get things up and running if you really care. I don't remember what problems you were having with installation.